so_attributes.hpp

```diff
@@ -124,7 +124,7 @@
 #       if defined LMI_MINGW_W64
 #           define LMI_SO_FWD_DECL
 #       else  // !defined LMI_MINGW_W64
-#           define LMI_SO_FWD_DECL LMI_SO
+#           define LMI_SO_FWD_DECL
```

Comment:
We probably should just get rid of this one completely... but let's leave it for now.